auditbeat/module/file_integrity/kprobes/seccomp_linux_amd64.go

```diff
@@ -18,27 +18,21 @@
 package kprobes
 
 import (
-	"runtime"
-
 	"github.com/elastic/beats/v7/libbeat/common/seccomp"
 )
 
 func init() {
-	switch runtime.GOARCH {
-	case "amd64", "386", "arm64":
```

Comment:
You had customization for 386. Confirming your intention to remove this? Beats 8.0+ officially (per https://www.elastic.co/support/matrix) do not support 32-bit platforms.

Reply:
thanks for asking for confirmation on that one; yes this is intentional as the BTF files we base  upon (through `tk-btf`) for the kprobes FIM BE do provide offsets only for `amd64` and `arm64`. As a result, kprobes BE supports only these two archs (IIRC the same applies to ebpf BE)